flang/README.md

```diff
@@ -8,34 +8,26 @@
 
 # FIR
 
-Working branch for FIR development.
+Yes, we've moved. This is now the working branch for FIR development.
```

Comment:
is `fir-dev` going to be the default branch one sees in the `f18-llvm-project` repository? If not we don't need to have `Yes, we've moved` here, do we?

Reply:
It's not the default branch.

Reply:
I am planning to update https://github.com/schweitzpgi/f18/tree/mono to point here. The "we've moved" was intended to be a visual clue that people have gone to the correct place.

Reply:
I conflated this move with the f18 move to llvm monorepo. Disregard my comment.